ui/packages/atlasmap/src/Atlasmap/dialogs/usePropertyDialog.tsx

```diff
@@ -13,9 +13,10 @@ export function usePropertyDialog(
   const [onPropertyCb, setOnPropertyCb] = useState<PropertyCallback | null>(
     null,
   );
-  const [initialProperty, setInitialProperty] = useState<IProperty | null>(
-    null,
-  );
+  const [initialProperty, setInitialProperty] = useState<IProperty | null>({
+    valueType: propertyTypes[0][0],
+    scope: getPropertyScopeOptions()[0].value,
```

Comment:
Added also the initial `scope`. This didn't cause error mentioned in [ENTESB-14445](https://issues.redhat.com/browse/ENTESB-14445), but omitting this caused, that in case the initial value wasn't changed, the user would expect it was set to "Current Message Header" which was visible in the dropdown, but was actually empty string in the mapping file